### PR TITLE
Trophic levels biomass mjg

### PR DESCRIPTION
**Pull request by DrMattG**
Updated James' pull request

### CODE REVIEW

**Comment by DrMattG** [comment]
This superceeds the previous pull requests

File: trophic_levels-biomass_indicators.rmd [hunk 1]
```diff
@@ -210,11 +210,11 @@ totalcarnivores<-carnivorerast[[49:60]]
```
Comment (HannoSandvik) [inline trophic_levels-biomass_indicators.rmd:210]: Here the years are not visible in the headings of the 12 panels. Would be great to see them.

**Review by HannoSandvik — CHANGES_REQUESTED** [review]
I re-inserted by previous suggestions. Now I am fine with this version, except for my comment on the missing labels in the panels of one figure.

**Comment by DrMattG** [comment]
Merging this now with changes to the plots suggested by @HannoSandvik

**Pull request merged**